src/Mod/Sketcher/App/GeometryFacade.h

```diff
@@ -118,8 +118,9 @@ class SketcherExport GeometryFacade: public Base::BaseClass, private ISketchGeom
     friend class GeometryFacadePy;
 
 public:  // Factory methods
-    static std::unique_ptr<GeometryFacade> getFacade(Part::Geometry* geometry, bool owner = false);
-    static std::unique_ptr<const GeometryFacade> getFacade(const Part::Geometry* geometry);
+    static std::unique_ptr<GeometryFacade> getFacade(const Part::Geometry* geometry,
+                                                     bool owner = false);
+    // static std::unique_ptr<const GeometryFacade> getFacade(const Part::Geometry* geometry);

```

Comment:
Same as previous - if it is commented for a reason it should be stated somewhere.